src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAvrcpControl.cs

```diff
@@ -0,0 +1,416 @@
+                    ConnStateChanged?.Invoke(this, e);
+                }
+                else
+                {
+                    _taskForDisconnection.SetException(BluetoothErrorFactory.CreateBluetoothException((int)BluetoothError.OperationFailed));
+                }
+                _taskForDisconnection = null;
+            }
+        }
+
+        /// <summary>
+        /// The AvrcpControlConnectionChangedEventArgs event is invoked when the connection status of device is changed.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public event EventHandler<AvrcpControlConnectionChangedEventArgs> ConnStateChanged;
+
+        /// <summary>
+        /// The PositionChangedEventArgs event is invoked when the play position of a track is changed.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public event EventHandler<PositionChangedEventArgs> PositionChanged;
+
+        /// <summary>
+        /// The PlayStateChangedEventArgs event is invoked when the play state of a track gets changed.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public event EventHandler<PlayStateChangedEventArgs> PlayStateChanged;
+
+        /// <summary>
+        /// The TrackInfoChangedEventArgs event is invoked when info of a track gets changed.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public event EventHandler<TrackInfoChangedEventArgs> TrackInfoChanged;
+
+        /// <summary>
+        /// Asynchronously connects the remote device
+        /// </summary>
+        /// <param name="remoteAddress">Address of device to be connected</param>
+        /// <exception cref="NotSupportedException">Thrown when the Bluetooth is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method fails</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public Task ConnectAsync(string remoteAddress)
+        {
+            if (_taskForConnection != null && !_taskForConnection.Task.IsCompleted)
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NowInProgress);
+            }
+
+            _taskForConnection = new TaskCompletionSource<bool>();
+            _remoteAddress = remoteAddress;
+            BluetoothAvrcpControlImpl.Instance.Connect(remoteAddress);
+            return _taskForConnection.Task;
+        }
+
+        /// <summary>
+        /// Asynchronously disconnects the remote device
+        /// </summary>
+        /// <param name="remoteAddress">Address of device to be disconnected</param>
+        /// <exception cref="NotSupportedException">Thrown when the Bluetooth is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method fails</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public Task DisconnectAsync(string remoteAddress)
```

Comment:
I wonder why need `remoteAddress`, It seems not allow multiple connections, so disconnect remote address always a remote address that used on `ConnectAsync`,

Reply:
```suggestion
        public Task DisconnectAsync()
        {
          ...
          BluetoothAvrcpControlImpl.Instance.Disconnect(_remoteAddress);
          ...
        }
```

Reply:
Done.

Reply:
Thank you. disconnect remote address always a remote address used on ConnectAsync.